launcher/launcher.go

```diff
@@ -81,10 +84,28 @@ func (l *launcher) Run(b *beat.Beat) error {
 	if err := b.Manager.Start(); err != nil {
 		return err
 	}
-	defer b.Manager.Stop()
+
+	if !b.Manager.Enabled() {
+		sigc := make(chan os.Signal, 1)
+		signal.Notify(sigc, syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
+		var sigCount = 0
+		go func() {
+			for range sigc {
+				sigCount++
+				if sigCount >= 2 {
+					l.log.Infof("Received second signal, terminating immediately")
+					os.Exit(1)
+				}
+			}
+
+		}()
+	}
```

Comment:
this is intended to run only on standalone cloudbeat and enable support for exiting immediately after a double SIGINT 

note the we [won't](https://github.com/elastic/elastic-agent-libs/blob/main/service/service.go#L46-L55) be notified by a second signal

lmk if this should be kept in the PR or not